adapter/mysql/mysql.go

```diff
@@ -78,6 +78,10 @@ func (d *MysqlDatabase) Views() ([]string, error) {
 	return ddls, nil
 }
 
+func (d *MysqlDatabase) Constraints(table string) ([]*adapter.ColumnConstraints, error) {
```

Comment:
If you followed the current design https://github.com/k0kubun/sqldef/pull/121#discussion_r649057839, I don't think you needed to add this API to other adapters.